ruby/vampires.rb

```diff
@@ -0,0 +1,75 @@
+# method to determine the likelihood of being a vampire
+def determine_vampire (likes_garlic, needs_insurance, age_check, name)
```

Comment:
Careful with methods that take more then 3 arg, what if it needed 10?  4 is a good point to see if things can be separated more.